File: 2st/Riyeon/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        String a = "", b="";
        for(int i : num_list){
            if(i%2!=0)
                a += String.valueOf(i);
            else
                b += String.valueOf(i);
        }
        return Integer.parseInt(a)+Integer.parseInt(b);
    }
}

File: 4st/Seohyun/programmer/이어 붙인 수.js
class Solution {
    public int solution(int[] num_list) {
        StringBuilder sb1 = new StringBuilder();
        StringBuilder sb2 = new StringBuilder();
        
        for(int i=0; i<num_list.length; i++){
            if(num_list[i]%2==0){//짝수
                sb1.append(num_list[i]);
            }else{
                sb2.append(num_list[i]);
            }
        }
        
        int n1 = Integer.parseInt(sb1.toString());
        int n2 = Integer.parseInt(sb2.toString());
        
        return n1 + n2;
    }
}